src/bicep/mlz.bicep

```
@@ -205,7 +208,26 @@ param identityVirtualNetworkDiagnosticsLogs array = []
 param identityVirtualNetworkDiagnosticsMetrics array = []
 
 @description('An array of Network Security Group Rules to apply to the Identity Virtual Network. See https://docs.microsoft.com/en-us/azure/templates/microsoft.network/networksecuritygroups/securityrules?tabs=bicep#securityrulepropertiesformat for valid settings.')
-param identityNetworkSecurityGroupRules array = []
+param identityNetworkSecurityGroupRules array = [
+  {
+    name: 'Identity-East-West'
```

nit naming: For this and the other two rules, should we just name them for what they do?

(plus, now that we generate resource names on the user's behalf [a user may modify the template to describe their spokes with whatever deployment-time name they like](https://github.com/Azure/missionlz/blob/b7e1d7e87ba94896f7f94e16afa66481672ef451/src/bicep/mlz.bicep#L497) "Identity" in the name might end up inaccurate)

```suggestion
    name: 'Allow-Traffic-From-Spokes'
```